sources/packages/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Query,
+} from "@nestjs/common";
+import {
+  ApplicationService,
+  EducationProgramOfferingService,
+  InstitutionLocationService,
+  EducationProgramService,
+} from "../../services";
+import BaseController from "../BaseController";
+import {
+  GetApplicationBaseDTO,
+  transformToApplicationDto,
+  StudentApplicationAndCount,
+} from "./models/application.model";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { Application } from "../../database/entities";
+import {
+  FieldSortOrder,
+  DEFAULT_PAGE_NUMBER,
+  DEFAULT_PAGE_LIMIT,
+  transformToApplicationSummaryDTO,
+  getOfferingNameAndPeriod,
+} from "../../utilities";
+import { ApiNotFoundResponse, ApiOkResponse, ApiTags } from "@nestjs/swagger";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("application")
+@ApiTags("application")
```

Comment:
Can we start to add the tags as below for now on the controller that are client-specific?
This will bring some grouping while we do not have a final solution.
```typescript
@ApiTags(`${ClientTypeBaseRoute.SystemAccess}-application`)
```
@guru-aot @dheepak-aot @henryoforeh-aot, thoughts?

Reply:
![image](https://user-images.githubusercontent.com/77353155/158900109-b885eff5-617f-4aae-89b9-c1263556f9e3.png)
updated